presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/EnvironmentOptions.java

```diff
@@ -29,16 +29,14 @@
     @Option(name = "--server-package", title = "server-package", description = "path to Presto server package")
     public File serverPackage = new File("presto-server/target/presto-server-${project.version}.tar.gz");
 
-    @Option(name = "--environment", title = "environment", description = "the name of the environment to start", required = true)
-    public String environment;
+    @Option(name = "--environment", title = "environment", description = "the name of the environment to start", required = false)
+    public String environment = "";
```

Comment:
This _is_ a required for `env up`.
It's _not_ required for `env list`.
So it should not be in the options class shared between the two.